Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -14,6 +14,21 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+
+/**
+ Possible values for 3DS source support
+
+ - STPThreeDSecureSupportTypeDisabled: 3DS sources will  never be created
+ - STPThreeDSecureSupportTypeStatic: An attempt to create 3DS sources will be done for all card sources that may possibly support 3ds
+   A failure to create a 3DS source for a card that does not require 3DS will result in the original card source being passed through 
+   to your backend API adapter for charging. If you want to forbid all non-3DS payments, you should not charge and throw an error
```

Comment:
the original card source gets returned in `didCreatePaymentResult:` – PaymentContext doesn't actually hook into your backend API adapter to tell it to charge.